apps/studio/components/interfaces/Organization/Usage/Usage.utils.ts

```diff
@@ -17,7 +18,16 @@ export const generateUsageData = (attribute: string, days: number): DataPoint[]
   })
 }
 
-export const getUpgradeUrl = (slug: string, subscription?: OrgSubscription, source?: string) => {
+export function useGetUpgradeUrl(slug: string, subscription?: OrgSubscription, source?: string) {
+  const { billingAll } = useIsFeatureEnabled(['billing:all'])
+
+  if (!billingAll) {
+    const subject = `Enquiry to upgrade plan for organization`
+    const message = `Organization Slug: ${slug}\nRequested plan: <Specify which plan to upgrade to: Pro | Team | Enterprise>`
+
+    return `/support/new?slug=${slug}&projectRef=no-project&category=Plan_upgrade&subject=${subject}&message=${encodeURIComponent(message)}`
```

Comment:
might be something in the support form but the slug doesn't get chosen correctly

Reply:
ohh needs `orgSlug`